spirv/Cargo.toml

```diff
@@ -11,9 +11,14 @@ readme = "README.md"
 license = "Apache-2.0"
 keywords = ["spirv", "definition", "struct", "enum"]
 
+[features]
+serialize = ['serde']
+deserialize = ['serde']
+
 [lib]
 path = "lib.rs"
 
 [dependencies]
 bitflags = "1"
 num-traits = "0.2"
+serde = {version = "1", optional = true, features = ["derive"]}
```

Comment:
note that this isn't correct, feature should be `serde_derive`, or the version needs to be more specific